README.md

```diff
@@ -19,6 +19,7 @@
 - [Tiếng Việt - Vietnamese](translations/README-vi.md)
 - [Español](translations/README-es.md)
 - [Português Brasileiro](translations/README-ptbr.md)
+- [Polish](https://github.com/mbiesiad/coding-interview-university/blob/master/translations/README-pl.md)
```

Comment:
You can remove the full URL and just do translations/README-pl.md

Reply:
Sure, done. Thank you!